src/cljs/athens/views/appearance_settings/core.cljs

```diff
@@ -0,0 +1,154 @@
+   [:path {:d "M3,12 H16"}]])
+
+
+(def full-width-icon
+  [:svg {:viewBox "0 0 24 24"}
+   width-background
+   [:path {:d "M2,04 H22"}]
+   [:path {:d "M2,08 H22"}]
+   [:path {:d "M2,12 H8"}]])
+
+
+;; Style
+
+(def dropdown-style
+  {::stylefy/manual [[:.menu {:background (color :background-plus-2)
+                              :color (color :body-text-color)
+                              :border-radius "calc(0.25rem + 0.25rem)" ; Button corner radius + container padding makes "concentric" container radius
+                              :padding "0.25rem 0"
+                              :display "inline-flex"
+                              :box-shadow [[(:64 DEPTH-SHADOWS) ", 0 0 0 1px rgba(0, 0, 0, 0.05)"]]}]]})
+
+
+(def preferences-set-style
+  {:display "grid"
+   :grid-auto-flow "column"
+   :grid-gap "1px"
+   :grid-auto-columns "1fr"
+   :border-radius "calc(0.25rem + 1px)"
+   :margin "0.125rem 0.5rem"
+   :padding "1px"
+   :box-shadow "inset 0 0 0 1px transparent, 0 0 0 1px transparent"
+   :transition "box-shadow 0.1s ease, filter 0.1s ease"
+   :background "inherit"
+   ::stylefy/manual [[:&:hover {:filter "brightness(120%)"
+                                :box-shadow [["0 0 0 1px " (color :border-color)]]}]
+                     [:>button {:text-align "center"
+                                :display "flex"
+                                :gap "0.25rem"
+                                :text-transform "uppercase"
+                                :flex-direction "column"
+                                :align-items "center"
+                                :justify-content "center"
+                                :background "inherit"
+                                :min-height "2.5rem"
+                                :padding "0.5rem"
+                                :font-weight "500"}
+                      [:span {:font-size "85%"
+                              :color (style/color :body-text-color :opacity-med)}]
+                      [:svg {:vector-effect "non-scaling-stroke"
+                             :overflow "visible"
+                             :width "24px"
+                             :height "24px"}]
+                      ["svg:not(.MuiSvgIcon-root)" {:fill "none"
+                                                    :stroke-linecap "round"
+                                                    :stroke "currentColor"
+                                                    :stroke-width "0.12rem"}]]]})
+
+
+;; Components
+
+(defn preferences-set
+  [{:keys [prefs current]}]
+  [:div (use-style preferences-set-style)
+   (doall (for [option prefs]
+            [button {:class (:id option)
+                     :key (:id option)
+                     :active (= (:id option) current)
+                     :on-click (:fn option)}
+             (:content option)]))])
+
+
+(def theme-settings
+  [{:content [:<> [:> Brightness7] [:span "Light"]]
+    :id "theme-light"
+    :fn #(dispatch [:theme/set-light])}
```

Comment:
maybe you could have just one event, and give it an argument saying which theme to use?

e.g.: `(dispatch [:theme/set :dark])`

also, maybe it might make sense to rename the event & local storage keys to `appearance/...`, theme is an aspect of appearance.

Reply:
oh, I just remembered Athens already has themes so changing the key for themes would be annoying. disregard that idea.